src/PreviewPanel.tsx

```diff
@@ -1,27 +1,24 @@
 import * as React from "react";
-import { useArgs, useParameter, useChannel } from "@storybook/api";
+import { useArgs, useParameter, useChannel, useGlobals } from "@storybook/api";
 import { Tab, Tabs, TabList, TabPanel } from "react-tabs";
-import * as Prism from "prismjs";
-
-import "prismjs/components/prism-typescript";
```

Comment:
Do we still have a direct dependency to prism ?
If no we can remove prism from the package.json
https://github.com/naver/storybook-addon-preview/blob/master/package.json#L33-L40

Reply:
@ryuran 

Removed prism.
Test storybook-addon-preview@2.3.0-beta.1